docs/tutorials/tutorial2.md

```diff
@@ -14,148 +14,49 @@ As reported in our paper, we trained an AtacWorks model to learn a mapping from
 
 If you want to train your own AtacWorks model instead of using the model reported in the paper, refer to [Tutorial 1](tutorial1.md).
 
+## Downloading pre-trained models
 
-## Step 1: Create folder and set AtacWorks path
+All models described in [Lal & Chiang, et al. (2019)](https://www.biorxiv.org/content/10.1101/829481v2) are available for download and use at `https://atacworks-paper.s3.us-east-2.amazonaws.com`. 
```

Comment:
Are we going to replace this with NGC registry?

Reply:
Yes, once the models go LIVE. For now, this link works, so I'd like to keep it this way until NGC models are published